src/main/java/core/basesyntax/model/Transaction.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Transaction {
+    private Operation abbreviature;
+    private Fruit fruit;
+    private Integer quantity;
+
+    public Transaction(Operation operation, Fruit fruit, Integer quantity) {
+        this.abbreviature = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getAbbreviature() {
+        return abbreviature;
+    }
+
+    public Fruit getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(Fruit fruit) {
+        this.fruit = fruit;
+    }
+
+    public Integer getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(Integer quantity) {
+        this.quantity = quantity;
+    }
+
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private String operation;
+
+        Operation(String operation) {
+            this.operation = operation;
+        }
+
+        public String getOperation() {
+            return operation;
+        }
```

Comment:
remove this method, you don't need it

Reply:
This method is used in AvailableOperationsMap